gapic-generator-java-pom-parent/pom.xml

```diff
@@ -27,7 +27,7 @@
         consistent across modules in this repository -->
     <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
     <grpc.version>1.60.0</grpc.version>
-    <google.auth.version>1.20.0</google.auth.version>
+    <google.auth.version>1.21.0</google.auth.version>
```

Comment:
Manually update as renovate bot has not picked up the changes yet.

Reply:
Let's create a separate PR to update this dependency, so it can properly show up in release notes.